cmd/otelcorecol/go.mod

```diff
@@ -84,7 +84,7 @@ require (
 	go.opentelemetry.io/collector/config/confighttp v0.91.0 // indirect
 	go.opentelemetry.io/collector/config/confignet v0.91.0 // indirect
 	go.opentelemetry.io/collector/config/configopaque v0.91.0 // indirect
-	go.opentelemetry.io/collector/config/configretry v0.91.0 // indirect
+	go.opentelemetry.io/collector/config/configretry v0.0.0-20231221085427-9027a8d9cc3f // indirect
```

Comment:
You cannot do this when you add the module. You need a second PR for doing this.

Reply:
yup... that's right, it needs to be a follow up. i usually leave the module version a v0.0.0 to remember 😆